src/openrct2/ride/RideRatings.cpp

```diff
@@ -95,7 +95,61 @@ static void ride_ratings_begin_proximity_loop(RideRatingUpdateState& state);
 static void RideRatingsCalculate(RideRatingUpdateState& state, Ride& ride);
 static void RideRatingsCalculateValue(Ride& ride);
 static void ride_ratings_score_close_proximity(RideRatingUpdateState& state, TileElement* inputTileElement);
-static void ride_ratings_add(RatingTuple* rating, int32_t excitement, int32_t intensity, int32_t nausea);
+static void RideRatingsAdd(RatingTuple* rating, int32_t excitement, int32_t intensity, int32_t nausea);
+
+static ShelteredEights get_num_of_sheltered_eighths(const Ride& ride);
+static money64 ride_compute_upkeep(RideRatingUpdateState& state, const Ride& ride);
+static void SetUnreliabilityFactor(Ride& ride);
+
+static void RideRatingsApplyAdjustments(const Ride& ride, RatingTuple* ratings);
+static void RideRatingsApplyIntensityPenalty(RatingTuple* ratings);
```

Comment:
```suggestion
static void RideRatingsApplyIntensityPenalty(RatingTuple& ratings);
```
I believe this is not supposed to be optional, all the other functions should also do this.